Modules/StudyProgramme/classes/types/class.ilStudyProgrammeTypeGUI.php

```diff
@@ -27,11 +27,12 @@ class ilStudyProgrammeTypeGUI
     protected ilStudyProgrammeTypeRepository $type_repository;
     protected Input\Factory $input_factory;
     protected Renderer $renderer;
-    protected ServerRequest $request;
+    protected Psr\Http\Message\ServerRequestInterface $request;
```

Comment:
Can we get rid of this completely in favour of `ILIAS\HTTP\Wrapper\RequestWrapper`?

Reply:
Not yet, the 'withRequest' method typehints on that type.